### PR TITLE
Orders

### PR DESCRIPTION
**Pull request by davin01001**
operations with orders

### CODE REVIEW

File: functions/src/api/orders/controllers/orders/create-order.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { Response } from "express";
+// import * as admin from "firebase-admin";
```
Comment (pavelsokolov21) [inline functions/src/api/orders/controllers/orders/create-order.ts]: !

File: functions/src/api/orders/controllers/orders/get-all-orders.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    const ordersRef = firestore.collection(DBCollections.ORDERS);
+    const docs = await ordersRef.get();
+    const orders: Order[] = [];
+    docs.forEach( (doc) => {
+      const order = doc.data() as Order;
+      order.id = doc.id;
+      orders.push(order);
+    });
+    // const ordersResponseData = orders
+    //   .filter( (order) => (order.status != ORDER_STATUSES.DECLINED
```
Comment (pavelsokolov21) [inline functions/src/api/orders/controllers/orders/get-all-orders.ts]: !

File: functions/src/api/orders/controllers/orders/get-order-by-id.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { Response } from "express";
+// import * as admin from "firebase-admin";
```
Comment (pavelsokolov21) [inline functions/src/api/orders/controllers/orders/get-order-by-id.ts]: !

File: functions/src/api/orders/controllers/orders/get-order-by-id.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import { DBCollections } from "../../../../enums";
+import { Order } from "../../../../interfaces";
+import { handleApiError } from "../../../../utils";
+import { RequestWithOrderId } from "./order-requests";
+
+export const getOrderById = async (req: RequestWithOrderId, res: Response) => {
+  try {
+    const { uid } = req.params;
+    // const user = await admin.auth().getUser(uid);
```
Comment (pavelsokolov21) [inline functions/src/api/orders/controllers/orders/get-order-by-id.ts]: !

File: functions/src/api/orders/controllers/orders/update-order.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import { Response } from "express";
+// import * as admin from "firebase-admin";
```
Comment (pavelsokolov21) [inline functions/src/api/orders/controllers/orders/update-order.ts]: !

File: functions/src/api/orders/controllers/orders/update-order.ts [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        departure,
+        destination,
+        language,
+        car,
+        price,
+        comment,
+      },
+      params: { uid },
+    } = req;
+    // const authInstance = admin.auth();
```
Comment (pavelsokolov21) [inline functions/src/api/orders/controllers/orders/update-order.ts]: !

File: functions/src/api/orders/controllers/orders/update-order.ts [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+    const firestore = req.firebase.firestoreInstance;
+
+    await firestore.collection(DBCollections.ORDERS).doc(uid).update({
+      name,
+      // date,
```
Comment (pavelsokolov21) [inline functions/src/api/orders/controllers/orders/update-order.ts]: !

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

**Comment by pavelsokolov21** [comment]
Удалить .idea

**Comment by davin01001** [comment]
agree with all, changed

**Review by pavelsokolov21 — APPROVED** [review]

**Pull request merged**